### PR TITLE
Dart

### PR DESCRIPTION
**Pull request by tectrixer**
working dart implementation

### CODE REVIEW

**Review by Zollerboy1 — COMMENTED** [review]
Looks good. Maybe the server should log that it started running.

**Review by Zollerboy1 — APPROVED** [review]
Nice!

**Comment by vypxl** [comment]
Looks good, but please remove the commented-out code in the yaml files.

**Review by vypxl — APPROVED** [review]

**Comment by tectrixer** [comment]
You think I should merge this, although this does not implement all of the actions (no UPLOAD)?

**Comment by vypxl** [comment]
The other implementations also don't atm, and the client does not test them so yeah

**Review by umgefahren — CHANGES_REQUESTED** [review]
Please include in the README that you added support for Delete

**Comment by tectrixer** [comment]
@umgefahren Now readme should satisfy your needs

File: Dart/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Dart implementation
+
+## Current supportet requests
+
+- `GET *<key>*`
+- `SET *<key>* *<value>*`
+- `GETC`
+- `SETC`
```
Comment (umgefahren) [inline Dart/README.md:8]: If your implementation implements `DELC` please specify that, if not, implement it.

File: Dart/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- `DEL *<key>*`
```
Comment (umgefahren) [inline Dart/README.md:9]: Reorder this to the 7th line

**Review by umgefahren — CHANGES_REQUESTED** [review]
Please update your README

**Review by umgefahren — APPROVED** [review]
Finnally 😉

**Pull request merged**